cfn-resources/third-party-integration/mongodb-atlas-thirdpartyintegration.json

```diff
@@ -33,6 +33,7 @@
         }
     },
     "primaryIdentifier": [
+        "/properties/ProjectId",
```

Comment:
I think would make more sense to use the profile (https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/220) and add projectId in the profile instead

Reply:
discussed this in the standup. We are okay with releasing the same cfn template multiple times. Closing this comment

Reply:
We should be able to use the profile for an override, but many of these commands will require the user to specify the parameters as part of the global config.

The profile should only be used in cases where the user has not specified a required param that can't be provided as part of the resource primary identifiers.